Section intro

StoneCat opened the pull request.

Commit:
Add and write section 'Vorwort' @bastelfreak todos

Commit:
Start writing section 'Einführung' Part 1

Commit:
Adding last Part of Section 'Intro'

Commit:
Include Vorwort in Einführung. Correct word spell

StoneCat commented:
@bastelfreak Change y requests in comment 6098501

Marcellii commented on thesis-de/content-de.tex:
```diff
@@ -16,18 +16,91 @@ \section{Erklärung}
+Selbsteinschätzung somit weiter fortgebildet oder sogar korrigiert.
+
+Großen Dank geht auch an die Kollegen von XYZ und an den Herrn XYZ welcher
+uns eine Lizens von JIRA bereitgestellt hat. Auch die Jungs von Gruppe ABC
```
das Auch die Jungs kann man hier nicht schreiben. Hier sollte @bastelfreak nochmal drüberschauen udn die nötigen Namen nennen können

bastelfreak replied:
Die Lizenz hat uns Atlassian für das Projekt gestellt

Marcellii commented on thesis-de/content-de.tex:
```diff
@@ -16,18 +16,91 @@ \section{Erklärung}
+Gleichzeitig beschäftigen sich Computersystem-Anbieter mit der optimalen
+Auslastung ihrer Serverfarmen (Hardware) und den damit verbundenen
+Kostenoptimierungen. Durch einen neu entstandenen Trend des Cloud Hostings
+ist die optimale Auslastung der Hardwaremittel wichtiger denn je. Jedoch
```
eventuell statt denn je > als sonst

Marcellii commented on thesis-de/content-de.tex:
```diff
@@ -16,18 +16,91 @@ \section{Erklärung}
+Rechenzentren für die Ausarbeitung einer Ressourcen-Übersicht. Diese
+soll primär von IT-Fachkräften einsehbar sein, sodass die Gesamtperformance
+des Rechenzentrums optimiert werden kann. Je nach Datenschutzeinwilligung
+der Kunden von den einzelnen Computersystem-Anbieter können diese Daten auch
```
Anbietern

Commit:
Correct word spelling by marcelli @ Einführung

bastelfreak commented on thesis-de/content-de.tex:
```diff
@@ -16,18 +16,91 @@ \section{Erklärung}
+Unterrichtszeit im dritten Semester. Herr Meusel und Herr Luis
+sprachen während einem Abendessen über die Inhalte ihres täglichen
+Arbeitstages. Herr Meusel arbeitete zu diesem Zeitpunkt an einem Projekt
+im Bereich Cloud Hosting und Herr Luis war von Datenanalysen und großen
```
Herr Luis war.. aufgeschlossen? komischer satzbau

bastelfreak commented on thesis-de/content-de.tex:
```diff
@@ -16,18 +16,91 @@ \section{Erklärung}
+definierten Vorgaben waren somit erfüllt und das Projekt konnte für das
+fünfte und sechste Semester angemeldet werden.
+
+Eine der größten Hürden in diesem Projekt war die fehlende Zeit für
```
Das ist ein rückblick, das gehört nicht in die einleitung, höchstens in das fazit

bastelfreak commented on thesis-de/content-de.tex:
```diff
@@ -16,18 +16,91 @@ \section{Erklärung}
+lassen und somit, wie auch bei einem zu langsamen Arbeitstempo, das Projektziel
+gefährdet.
+
+Bei der durchführung des Projektes wurde bei jedem Projektmitglied der
```
durchführung -> Durchführung

Commit:
Correct bastelfreak comment @ 28.01 14:11

Commit:
Correct bastelfreak comment #2 @ 28.01. 14:11
After talking with @bastelfreak over voice-chat this part must be written
in the future.
We can add in section 'Rückblick' a feedback about the content.

Commit:
Correct bastelfreak comment #3 @ 28.01 14:11
Corrext the word spelling @ one word.

bastelfreak commented on thesis-de/content-de.tex:
```diff
@@ -16,18 +16,91 @@ \section{Erklärung}
+
+Die Zusammensetzung dieses Teams entstand während der rugulären
+Unterrichtszeit im dritten Semester. Herr Meusel und Herr Luis
+sprachen während einem Abendessen über die Inhalte ihres täglichen
```
genetiv, dativ, keine ahnung

bastelfreak commented on thesis-de/content-de.tex:
```diff
@@ -16,18 +16,91 @@ \section{Erklärung}
+Selbsteinschätzung somit weiter fortgebildet oder sogar korrigiert.
+
+Großen Dank geht auch an die Kollegen von XYZ und an den Herrn XYZ welcher
+uns eine Lizens von JIRA bereitgestellt hat. Auch die Jungs von Gruppe ABC
```
lizens -> lizenz
